controllers/object_controls.go

```diff
@@ -1213,6 +1265,36 @@ func TransformToolkit(obj *appsv1.DaemonSet, config *gpuv1.ClusterPolicySpec, n
 			}
 		}
 	}
+
+	// Update the driver-root volume if a custom driver install directory is configured with the operator
+	if config.HostPaths.DriverInstallDir != "" && config.HostPaths.DriverInstallDir != DefaultDriverInstallDir {
+		for index, volume := range obj.Spec.Template.Spec.Volumes {
+			if volume.Name == "driver-root" {
+				obj.Spec.Template.Spec.Volumes[index].HostPath.Path = config.HostPaths.DriverInstallDir
+			}
+		}
+	}
+
+	// If a custom devPath is configured which is not equal to 'hostRoot/dev' or 'driverInstallDir/dev', make it available
+	// to the toolkit container under '/dev-root/dev'
+	devRoot := filepath.Dir(config.HostPaths.DevPath)
```

Comment:
Is `DevPath` `/` or `/dev`? Do we validate that it ends in `/dev`?

Reply:
As currently implemented, I am expecting `DevPath` to be a `/dev` filesystem. It would probably be a good idea to validate that it ends in `/dev` ...

Reply:
I have removed devPath from the ClusterPolicy API now. Consequently, this code has been removed.